src/python/system/shell.py

```diff
@@ -350,14 +350,23 @@ def get_interpreter(file_to_execute):
   }
 
   try:
-    return interpreters[os.path.splitext(file_to_execute)[1]]
+    interpreter = interpreters[os.path.splitext(file_to_execute)[1]]
   except KeyError:
     return None
 
+  # TODO(mbarbella): Remove this when fuzzers have been migrated to Python 3.
+  if (is_blackbox_fuzzer and interpreter == sys.executable and
+      environment.get_value('USE_PYTHON2_FOR_BLACKBOX_FUZZERS') and
+      environment.platform() != 'WINDOWS'):
```

Comment:
Can we remove this "environment.platform() != 'WINDOWS'", since we copy python2.exe in windows startup script (from @oliverchang )

Reply:
We still need it for now since the Python 2 bots would break in this case.

Reply:
Can we do this check instead:


```python
if (environment.get_value('USE_PYTHON2_FOR_BLACKBOX_FUZZERS') 
     and sys.version_info.major == 3):
  ...
```

This will make Windows Py 3 work. We should be using the `python2` binary explicitly only in the Python 3 case anyway.

Reply:
Very good point, I'll make this change in a new PR.